path/_separator.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// This module is browser compatible.
+
+// NOTE: This file exists to break circular referencecs; it should NOT be
+// imported directly, outside of the "std/path" module.
+
+import { isWindows } from "../_util/os.ts";
+import { sep as win32Sep } from "./win32.ts";
+import { sep as posixSep } from "./posix.ts";
+
+export const sep = isWindows ? win32Sep : posixSep;
+// @deprecated (will be removed after 0.187.0) use `sep`
+export const SEP = sep;
+export const sepPattern = isWindows ? /[\\/]+/ : /\/+/;
+// @deprecated (will be removed after 0.187.0) use `sepPattern`
+export const SEP_PATTERN = sepPattern;
```

Comment:
The outcome of the discussions was to keep using upper snake case for constants: https://github.com/denoland/manual/pull/625

Reply:
@nayeemrmn  good to know thank you. I will take a look and see if anything is necessary for this PR to continue or not. I will close it if I don't think anything is needed to change.